pkg/nanopb/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME=nanopb
+PKG_URL=https://github.com/nanopb/nanopb.git
+PKG_VERSION=be0dd4b3aacdc9c2bbe21568b6a37218f2bb959a
+PKG_LICENSE=zlib
+
+.PHONY: all
+
+RIOT_CFLAGS = $(INCLUDES) -std=c99
+
+TOOLCHAIN_FILE = $(PKG_BUILDDIR)/xcompile-toolchain.cmake
+
+PKG_DIR=$(CURDIR)
+PROTO_DIR=$(APPDIR)/proto
```

Comment:
"compiled" `.proto` files should go under `bin/`. Unfortunately, there is currently no place to put "target-independent" files (like generated files) so they will have to go under `bin/${BOARD}/proto_whatever`.